recipes/azure-storage-cpp/all/test_package/conanfile.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class AwsSdkCppTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package_multi"
+
+    def build(self):
+        cmake = CMake(self)
+        if not self.settings.compiler.cppstd:
+            cmake.definitions["CMAKE_CXX_STANDARD"] = 11
```

Comment:
This could be move to the cmake since it the test package

Reply:
will do